.github/workflows/build.yml

```diff
@@ -51,12 +51,15 @@ jobs:
       with:
         files: ./artifacts/coverage-reports/Polly.Core.Tests/Cobertura.xml,./artifacts/coverage-reports/Polly.Specs/Cobertura.xml
         flags: ${{ matrix.os_name }}
+        if-no-files-found: warn
```

Comment:
This is the default, so we can omit this unless you want it to not do anything or fail the build.

Reply:
Thanks, cleaned this up.